src/ScrumMaster/IO/NotifierOutput.php

```diff
@@ -28,14 +30,25 @@ public function write(array $results): void
 
     private function writeChannel(string $name, ChannelResult $result): void
     {
-        $this->output->writeln("# CHANNEL: {$name}");
+        $loader = new FilesystemLoader(__DIR__ . '/templates');
+        $twig = new Environment($loader);
```

Comment:
I wouldn't couple these classes here. Why not inject this via the constructor?

Reply:
Agreed. Thank you! :)